agent/functional_tests/tests/functionaltests_test.go

```diff
@@ -567,9 +567,12 @@ func TestAwslogsDriver(t *testing.T) {
 		})
 	}()
 
+	strs := strings.Split(*testTask.TaskArn, "/")
+	taskId := strs[len(strs)-1]
+
 	params := &cloudwatchlogs.GetLogEventsInput{
 		LogGroupName:  aws.String(awslogsLogGroupName),
-		LogStreamName: aws.String(containerId),
+		LogStreamName: aws.String(fmt.Sprintf("ecs-functional-tests/awslogs/%s", taskId)),
```

Comment:
I don't think that's strictly necessary; the functional tests here are testing the agent and ECS functionality; the defaulting behavior for container ID is tested in the log driver unit tests inside the Docker codebase.